prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -416,10 +432,8 @@ def prompt_processor() -> Any:
                     The email should be directly assignable to a string variable. \
                         No explanation is required. If you cannot extract the email, output "NA".'  # noqa
                 answer = run_completion(
-                    llm_helper=llm_helper,
-                    llm_li=llm_li,
-                    prompt=prompt,
-                    adapter_instance_id=adapter_instance_id,
+                    llm,
+                    prompt,
```

Comment:
Can we pass this as named args instead of positional args for better readability?

Reply:
Done